content/realtime/channels.textile

```diff
@@ -575,7 +595,7 @@ bq(definition#publish-data).
   csharp:   void Publish(string name, object data, Action<bool, ErrorInfo> callback = null, string clientId = null)
   objc,swift: publish(name: String?, data: AnyObject?, callback: (("ARTErrorInfo":/realtime/types#error-info?) -> Void)?)
 
-Publish a single message on this channel based on a given event name and payload. A <span lang="default">callback</span><span lang="java">listener</span> may optionally be passed in to this call to be notified of success <span lang="default">or failure</span><span lang="ruby"></span> of the operation. If the channel is @initialized@ (i.e. no attempt to attach has yet been made for this channel), then calling @publish@ will implicitly attach the channel.
+Publish a single message on this channel based on a given event name and payload. A <span lang="default">callback</span><span lang="java">listener</span> may optionally be passed in to this call to be notified of success <span lang="default">or failure</span><span lang="ruby"></span> of the operation. <span lang="default">If the channel is @initialized@ (i.e. no attempt to attach has yet been made for this channel), then calling @publish@ will implicitly attach the channel.</span><span lang="jsall,ruby,swift">When publish is called with this client library, it "won't attempt to implicitly atach to the channel":#transient-publish.</span>
```

Comment:
I realise you've done a great job at keeping track of which libs have implemented these features, but I am concerned that this is going to lead to a lot of issues down the line.  I would almost prefer that we instead document the behaviour by default as this is 1.1 docs, and have a caveat (perhaps a p.note or some unique styling) that indicates this feature is not yet supported in lib X.